server/src/domain/server-info/server-info.service.spec.ts

```diff
@@ -169,7 +169,10 @@ describe(ServerInfoService.name, () => {
         loginPageMessage: '',
         oauthButtonText: 'Login with OAuth',
         trashDays: 30,
-        mapTileUrl: 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
+        mapStyles: [
+          { name: 'Immich Light', theme: 'light', url: '/api/system-config/map-light' },
```

Comment:
I think this url should not include `/api` and that part should come from the client.

Reply:
What do you mean by "that part should come from the client"?